elastic_appsearch.theme.inc

```diff
@@ -132,4 +142,84 @@ function template_preprocess_engine(array &$variables) {
     ],
   ];
 
-}
\ No newline at end of file
+}
+
+/**
+ * Prepares variables for search_api_index templates.
+ *
+ * Default template: search-api-index.html.twig.
+ *
+ * @param array &$variables
+ *   An associative array containing:
+ *   - index: The search index to display.
+ */
+function template_preprocess_server(array &$variables) {
+  $server = $variables['server'];
+  $engines_in_sync = $server->getEngines();
+  $engines = $server->getClient()->listEngines();
+
+  $headers = [
+    'Engine ID',
+    'Type',
+    'Document Count',
+    'Is Sync',
+    'Datasources',
+    'Operations'
+  ];
+  $rows = [];
+
+  foreach ($engines['results'] as $engine) {
+    $is_sync = isset($engines_in_sync[$engine['name']]);
+    $rows[] = [
+      'name' => ($is_sync) ?
+      Link::fromTextAndUrl($engine['name'], $engines_in_sync[$engine['name']]->toUrl('canonical')) :
+      $engine['name'],
+      'type' => $engine['type'],
+      'count' => $engine['document_count'],
+      'is_sync' => ($is_sync) ? 'Yes' : 'No',
+      'datasources' => ($is_sync) ? implode(', ', $engines_in_sync[$engine['name']]->datasources()) : '',
+      'op' => (!$is_sync) ?
+      Link::fromTextAndUrl('Sync Now',
+        Url::fromRoute('entity.elastic_appsearch_engine.sync', ['elastic_appsearch_engine' => $engine['name']])
+      ) :
+      build_ajax_url($engine),
+    ];
```

Comment:
This piece is less readable. Please fix the indentation and newline like this to make it more readable: 

```php
$rows[] = [
  'name' => ($is_sync) ? Link::fromTextAndUrl($engine['name'], $engines_in_sync[$engine['name']]->toUrl('canonical')) : $engine['name'],
  'type' => $engine['type'],
  'count' => $engine['document_count'],
  'is_sync' => ($is_sync) ? 'Yes' : 'No',
  'datasources' => ($is_sync) ? implode(', ', $engines_in_sync[$engine['name']]->datasources()) : '',
  'op' => (!$is_sync) ? Link::fromTextAndUrl('Sync Now', Url::fromRoute('entity.elastic_appsearch_engine.sync', ['elastic_appsearch_engine' => $engine['name']])) : build_ajax_url($engine),
];
```

Reply:
this will cause sonarqube to add code smell, i will try taking different approach